src/com/sproutlife/model/GameModel.java

```diff
@@ -15,102 +15,121 @@
 import com.sproutlife.model.step.GameStep;
 import com.sproutlife.model.step.GameStepListener;
 
+/**
+ * This is a class used to manage the game model essentials.
+ *
+ * @author Alex Shapiro
+ */
 public class GameModel {
 
-    Echosystem echosystem;  
-
-    GameClock clock;
-
-    GameStep gameStep;
-
-    GameThread gameThread;
-
-    Settings settings;
-
-    Stats stats;   
-    
-    public GameModel(Settings settings, ReentrantReadWriteLock interactionLock) {
-        this.settings = settings;
-        this.clock = new GameClock();
-        echosystem = new Echosystem(clock);
-        gameStep = new GameStep(this);       
-        gameThread = new GameThread(this, interactionLock);
-        stats = new Stats(this);
-    }
-
-    public void performGameStep() {
-        incrementTime();
-        gameStep.perform();
-    }
-    /*
-    public GameStep getGameStep() {
-        return gameStep;
-    }
-    */
-    public Echosystem getEchosystem() {
-        return echosystem;
-    }
-
-    public Board getBoard() {
-        return echosystem.getBoard();
-    }          
-
-    public int getTime() {
-        return clock.getTime();
-    }
-
-    public GameClock getClock() {
-        return clock;
-    }
-
-    private void incrementTime() {
-        clock.increment();
-    }
-
-    public void resetGame() {
-        getEchosystem().resetCells();        
-        getEchosystem().pruneEmptyOrganisms();
-        getEchosystem().clearRetiredOrgs();
-        getStats().reset();
-        getClock().reset();
-        
-    }
-    
-    public Stats getStats() {
-        return stats;
-    }   
-
-    public Settings getSettings() {
-        return settings;
-    }
-    
-
-    public void setPlayGame(boolean playGame) {
-        if (playGame) {          
-            gameThread.setPlayGame(true);
-
-        } else {
-            gameThread.setPlayGame(false);            
-        }
-    }
-    
-    public GameThread getGameThread() {
-        return gameThread;
-    }
-    
-    public boolean getPlayGame() {
-        return gameThread.getPlayGame();
-    }
-
-    public void setGameStepListener(GameStepListener l) {
-        if (gameThread!=null) {
-            gameThread.setGameStepListener(l);
-            gameStep.setGameStepListener(l);
-        }
-    }
-
-    public void set(String s, Object o) {
-        getSettings().set(s,o);
-    }
+	private Echosystem echosystem;
```

Comment:
Also, can we use 4 spaces as indentation instead of tabs. If there is any inconsistency with the existing code base using tabs in places then let's change those to spaces as well.

Reply:
Pretty sure theres something wrong with git, as in eclips and atom I see the indentation with a single tab.

Reply:
I'd like to use spaces as indentation. Your PR definitely has tabs, which should be spaces. I made a couple of commits on master (two days ago) that fix pre-existing tabs, so I checked to make sure.